.github/workflows/docker-end-user.yml

```diff
@@ -11,6 +11,7 @@ on:
   schedule:
     # '*' is a special character in YAML, so string must be quoted
     - cron: "0 4 * * *"
+  workflow_dispatch: ~
```

Comment:
FYI, I believe that the ~ is actually unnecessary (I have workflow with empty `workflow_dispatch:` and that seems to work all the same).